src/components/TitleBar/AccountButton.scss

```
@@ -1,7 +1,9 @@
+@use '../../styles/tokens' as tk;
+
 // src/components/TitleBar/AccountButton.scss
 //
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)